R/cleanup.R

```diff
@@ -42,6 +42,7 @@
 ##'
 ##' @export
 ##' @examples
+##' message(sprintf("Running in %s", format(environment())))
```

Comment:
Accidental leftover?

Reply:
indeed, trying to find out what withr/devtools were thinking they had as their environment - turns out it was exactly what I expected them to have, but it just did not work